mne/viz/epochs.py

```diff
@@ -95,6 +99,13 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
     scale_vmin = True if vmin is None else False
     scale_vmax = True if vmax is None else False
     vmin, vmax = _setup_vmin_vmax(data, vmin, vmax)
+    if logscale:
+        if (data >= 0).all():
+            data[data == 0] = np.nan
+            np.log2(data, out=data)
+        else:
+            raise ValueError('Only Epochs with positive data can be'
```

Comment:
you don't need the `if`-`else` if you do the `ValueError` first. You'll need only an `if` in that case